needles/control_center_users_locked.json

```diff
@@ -12,9 +12,15 @@
     },
     {
       "": "Unlock button",
+<<<<<<< HEAD
```

Comment:
merge conflicts to be resolved

Reply:
What? git didn’t tell me about this! argh.

Reply:
It seems I have somehow completely messed up the rebase and need to re-test this all. Bear with me.

Reply:
OK, ready for review again. Sorry about that.